js/index.js

```diff
@@ -803,14 +808,21 @@ function changeStyles() {
 	const selected = Array.from(styleSelectEl.options).filter(
 		(option) => option.selected
 	);
-	const selectedString = selected.map((option) => option.value);
+	let selectedString = selected.map((option) => option.value);
+
+	if (selectedString.find((selected) => selected === "None")) {
+		selectedString = [];
+		Array.from(styleSelectEl.options).forEach((option) => {
+			if (option.value !== "None") option.selected = false;
+		});
+	}
 
-	selectedString != "None"
-		? localStorage.setItem("promptStyle", JSON.stringify(selectedString))
-		: localStorage.setItem("promptStyle", "[]");
+	localStorage.setItem("promptStyle", JSON.stringify(selectedString));
```

Comment:
Yeah, my naming conventions were not really clear... selectedString is actually an array of strings. This comparison may have worked due to automagic JavaScript casting, but I wouldn't bet on the consistency of that.

Reply:
This last commit now fixes this, but also clears all selections if "None" is ever selected, as well as adding a helpful check indicator on applied styles:
![image](https://user-images.githubusercontent.com/7552751/205426654-df3ecdb0-16e9-4a26-ac76-08b5168e4637.png)

None in action:

https://user-images.githubusercontent.com/7552751/205426710-71322f8e-b3f2-49b9-a96b-1522bf56869c.mp4